onebusaway-enterprise-webapp/src/main/java/org/onebusaway/enterprise/webapp/actions/api/model/StopResult.java

```diff
@@ -32,17 +34,39 @@
 
 	private List<RouteAtStop> routesAvailable;
 
+	private List<StopBean> stops;
+
+	private List<String> stopIdsForRoute = new ArrayList<>();
+
 	public StopResult(StopBean stop, List<RouteAtStop> routesAvailable) {
 		this.stop = stop;
 		this.routesAvailable = routesAvailable;
 	}
+
+	public StopResult(StopBean stop, List<RouteAtStop> routesAvailable, List<StopsForRouteBean> fullStopList) {
```

Comment:
I added a new signature, rather than changing the old one, because I wasn't sure if the old one was still being used in other locations.